probe_src/python/probe_py/manual/workflows.py

```diff
@@ -139,7 +235,16 @@ def create_processes(self) -> None:
                 inputs = [n for n in self.graph.predecessors(node) if isinstance(n, FileNode)]
                 outputs = [n for n in self.graph.successors(node) if isinstance(n, FileNode)]
 
-                if self.is_standard_case(node, inputs, outputs) :
+                # Skip any command that calls another script
```

Comment:
We need a more general way (in a different PR) of handing the case where process forks child processes than asking if the command was `sh`. There are many other commands that do this, `make`, `bash`, ..., any command _could_ do this.

We should analyze this by the tree structure (later on).